### ISSUE TITLE
Rounded max and min amounts from /oracle/prices/stats endpoint

### ISSUE THREAD

**Issue by pantyporowicz** [helium/blockchain-http#335]
Hi,
I noticed problem with max and min properties from /oracle/prices/stats endpoint. Why are they rounded? I think, these amounts should be returned as decimals.
![image](https://user-images.githubusercontent.com/78741013/133767576-c1e402d3-a3cf-4238-a3ab-d6d29710def4.png)

**Comment by madninja** [comment]
These are reported as HNT and not as bones since their value would be too large for json/javascript to be able to parse

**Comment by szygendab** [comment]
@madninja, can you please, elaborate more on your answer?

**Comment by szygendab** [comment]
I think @pantyporowicz meant to ask for unifying the format of decimal places for `min` and `max` to be equal to decimal places in `avg` field

**Comment by madninja** [comment]
The format is precise though. The 20.0 is exactly 20.0 HNT and is _not_ rounded

**Comment by szygendab** [comment]
@madninja, so how come `median`, and `avg` are resulting in such values as above?

**Comment by madninja** [comment]
> @madninja, so how come `median`, and `avg` are resulting in such values as above?
> 
> They should also be `20.0` and `20.0`

Because the average is rounded to as many decimals precision as you see there..

**Comment by szygendab** [comment]
@madninja , but an example from above should actually be:

`min` - 20.0
`max` - 20.0
`avg` - 20.0
`median` - 20.0

...

**Comment by madninja** [comment]
yeah, we'd need to look at all the actual values to see what happened there.. This is just using the Postgres avg function over time series data

**Comment by szygendab** [comment]
@madninja, do you have any updates on this issue?

**Comment by madninja** [comment]
Nope.. do you have any additional data by looking at the database?

**Comment by szygendab** [comment]
@madninja, we are using only helium API. We do not have access to your database. Any HTTP request ( [such as this one](https://api.helium.io/v1/oracle/prices/stats?max+time=2021-09-16T00:00:00.000Z&min_time=2021-09-15T00:00:00.000Z) )returns rounded values for `min` and `max` as in the first example written by @pantyporowicz.
